queries.sql

```diff
@@ -8,3 +8,41 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.40 AND 17.3;
+
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT species from animals; -- verify that change was made
+ROLLBACK;
```

Comment:
* Kindly  verify that the change was undone by adding this query ``SELECT species from animals;``